irods/session.py

```diff
@@ -360,7 +360,7 @@ def connection_timeout(self, seconds):
         elif seconds is None:
             pass
         else:
-            exc = ValueError("The iRODS connection_timeout must be given a positive value of type int or float.")
+            exc = ValueError("The iRODS connection_timeout must be assigned to a positive int or float with units of seconds, or to None which effectively disables the timeout.")
```

Comment:
- "assigned to" -> "assigned"
- "with units of seconds" -> "(representing seconds)"
    - Maybe we just remove that part since that is supposed to be part of the documentation

Reply:
Yeah, I think removing the seconds bit seems appropriate. This is more about the class of value being passed in here.

Reply:
Done.